test/libsolidity/smtCheckerTests/functions/function_call_does_not_clear_local_vars.sol

```diff
@@ -9,4 +9,5 @@ contract C {
     }
 }
 // ----
+// Warning: (99-103): Assertion checker does not yet support the type of this variable.
```

Comment:
I would have expected this warning to be removed. Can you explain this?

Reply:
True, that's weird

Reply:
Ok just checked the code, the message is a bit misleading there. The type is supported but not identifier `this`. Should change the message at that particular location.